src/workflows-app/utils/submission-utils.ts

```diff
@@ -565,3 +586,12 @@ export const getOutputTableData = (
     _.orderBy([({ [sort.field]: field }) => _.lowerCase(field)], [sort.direction])
   )(configuredOutputDefinition);
 };
+
+export const fetchMetadata = async (
+  cromwellProxyUrl: string,
+  workflowId: string,
+  signal: AbortSignal,
+  includeKey: string[],
+  excludeKey: string[]
```

Comment:
Whenever a function accumulates a large number of arguments, consider putting some or all of them under an options object to make it clearer what's what when reading the function call.

https://www.codereadability.com/what-are-javascript-options-objects/